DATA/production/qc-async/mch-tracks.json

```diff
@@ -33,7 +33,7 @@
                 "maxNumberCycles": "-1",
                 "dataSource": {
                     "type": "direct",
-                    "query": "trackMCH:MCH/TRACKS;trackMCHROF:MCH/TRACKROFS;trackMCHTRACKCLUSTERS:MCH/TRACKCLUSTERS"
+                    "query": "trackMCH:MCH/TRACKS;trackMCHROF:MCH/TRACKROFS;trackMCHTRACKCLUSTERS:MCH/TRACKCLUSTERS;mchtrackdigits:MCH/CLUSTERDIGITS"
```

Comment:
It should be `;mchtrackdigits:MCH/TRACKDIGITS` instead of `;mchtrackdigits:MCH/CLUSTERDIGITS`

Reply:
Thanks!! Can you check now, @aferrero2707 ?

Reply:
Thanks, I think I took this from the sync-qc, maybe the wrong place, though (but the label was the correct one). Merging now.